developer-docs-site/docs/reference/glossary.md

```diff
@@ -236,6 +236,11 @@ then there is a guarantee that T_N will never be included in the blockchain.
 
 # R
 
+## Resource Account
+
+- A resource account can be created only once. An entity may call `create_account` in an attempt to claim an account ahead of the creation of a resource account. But if a resource account is found, Aptos will transition ownership of the account over to the resource account. This is done by validating that the account has yet to execute any transactions and that the `Account::signer_capbility_offer::for` is none. The probability of a collision where someone has legitimately produced a private key that maps to a resource account address is improbably low.
+- See the "resource account" references in [account.move](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/account.move) for more details.
```

Comment:
When I go over the glossary, first thing I want to know is "What is a resource account and how is it different from other accounts?" We should add a sentence or two explaining it. Rest of the above can stay or go from the glossary.

Reply:
> When I go over the glossary, first thing I want to know is "What is a resource account and how is it different from other accounts?" We should add a sentence or two explaining it. Rest of the above can stay or go from the glossary.

 Yeah, sorry about that. I should have marked this a draft for that first commit as I was iterating in a text file to begin with and left some of the lead out.

 As you can see, this should be addressed.

Reply:
Also, @gregnazario please help me understand if this lengthy description would be better served on another page. I'm still not clear where in our doc flow it comes up. Thanks!